tests/test___main__.py

```diff
@@ -1,11 +1,12 @@
 from __future__ import unicode_literals
 
-from mock import Mock, call
+from mock import Mock, call, patch
 
 from pg2kinesis.__main__ import Consume
 
 def test_consume():
```

Comment:
You mentioned this test was flaky but not sure what you changed to make it less so. I see the extra tests which are always welcome!.

Reply:
```
mock_formatter.cur_xact = "TEST_TRANSACTION"
```
This addition is what fixes the flakey test. The problem is that without this it's a "Mock" object, which cannot be formatted with special options (like the right align). This is only problematic if `time.time() % 10 == 0` given the way the logging is setup.

Reply:
added a comment to make it clear.